### ISSUE TITLE
Support for cloudwatch datasource

### ISSUE THREAD

**Issue by xocasdashdash** [K-Phoen/grabana#220]
I'm not 100% sure but am i right in assuming that as of today, Cloudwatch datasource will not work as is? 

what would it take to support it? i'm guessing i could follow one of the current datasources as an example to implement this. Does it make sense?

**Comment by K-Phoen** [comment]
You're right. As things are, Cloudwatch isn't supported. Looking at other datasources should give you a pretty good idea of what's necessary. Feel free to open a PR at any time, even in draft state, and I'll give you pointers if needed :)
